.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - 2.7
+  - 3.4
+  - 3.5
+addons:
+  postgresql: "9.3"
+install:
+  - pip install -U pip
+  - pip install -i https://pypi.pacificclimate.org/simple/ -r requirements.txt -r test_requirements.txt
```

Comment:
We should decide whether we are going to have a separate `test_requirements.txt`. Currently test requirements are in both files.